### PR TITLE
Rechecked the English and Bangla Translations

### PR DESCRIPTION
**Pull request by itz-rj-here**
## Description
Rechecked the English and Bangla Translations. Also made the English translation code more better to understand.

## Preview
image

## Checklist
- [x] I have read and followed the [Contributing Guidelines](../CONTRIBUTING.md).
- [x] I have added necessary tests.
- [x] The project builds and runs without issues.

### CODE REVIEW

**Comment by itz-rj-here** [comment]
@prem-k-r Requesting to check the changes

File: locales/bn.js [hunk 1]
```diff
@@ -20,108 +20,113 @@ const bn = {
...
+    "months": ['জানুয়ারি', 'ফেব্রুয়ারি', 'মার্চ', 'এপ্রিল', 'মে', 'জুন', 'জুলাই', 'আগস্ট', 'সেপ্টেম্বর', 'অক্টোবর', 'নভেম্বর', 'ডিসেম্বর'],
+    // "months": ['জানু', 'ফেব্রু', 'মার্চ', 'এপ্রি', 'মে', 'জুন', 'জুলাই', 'আগ', 'সেপ্টে', 'অক্টো', 'নভে', 'ডিসে'],   // Truncated
+
+    // Weather
+    "humidityLevel": "আর্দ্রতা",
+    "feelsLike": "অনুভূতি হয়",
+    "location": "পৃথিবী",
+
+    // Bookmarks
+    "bookmarksHover": "বুকমার্কস",
```
Comment (prem-k-r) [inline locales/bn.js:86]: have removed this bookmarksHover in #365 
using bookmarksHeading instead

Comment (itz-rj-here) [reply locales/bn.js:86]: ok

File: locales/en.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
     "menuCloseText": "Close",
+
     // Shortcuts
     "shortcutsText": "Shortcuts",
     "enableShortcutsText": "Show saved shortcuts",
     "editShortcutsText": "Edit Shortcuts",
     "shortcutsInfoText": "Choose which shortcuts get shown",
     "editShortcutsList": "Saved Shortcuts",
-    'editShortcutsListInfo': 'You can add new shortcuts by clicking the "+" icon or edit existing ones by clicking on the shortcut name or URL.',
+    "editShortcutsListInfo": "You can add new shortcuts by clicking the "+" icon or edit existing ones by clicking on the shortcut name or URL.",
```
Comment (prem-k-r) [inline locales/en.js:14]: use escape character `\"+\"`

Comment (itz-rj-here) [reply locales/en.js:14]: Uhh.?
okay

File: locales/bn.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const bn = {
     "editShortcutsText": "শর্টকাট সম্পাদনা করুন",
     "shortcutsInfoText": "যে শর্টকাটগুলো দেখাতে চান তা নির্বাচন করুন",
     "editShortcutsList": "সংরক্ষিত শর্টকাট",
-    "editShortcutsListInfo": "'+' আইকনে ক্লিক করে নতুন শর্টকাট যোগ করুন অথবা বিদ্যমান শর্টকাটের নাম বা URL এ ক্লিক করে সম্পাদনা করুন।",
+    "editShortcutsListInfo": ""+" আইকনে ক্লিক করে নতুন শর্টকাট যোগ করুন অথবা বিদ্যমান শর্টকাটের নাম বা URL এ ক্লিক করে সম্পাদনা করুন।",
```
Comment (prem-k-r) [inline locales/bn.js:14]: here too else + wont be visible.. pls check

Comment (itz-rj-here) [reply locales/bn.js:14]: Okkk

**Comment by prem-k-r** [comment]
seems fine

**Review by prem-k-r — APPROVED** [review]

**Comment by prem-k-r** [comment]
not changed any translation, right?
if yes, then will update the same in chrome branch pr

**Comment by itz-rj-here** [comment]
> not changed any translation, right? if yes, then will update the same in chrome branch pr

Yup. Just placed this changes as same as the english code.

**Comment by itz-rj-here** [comment]
@XengShi I think you should merge this.

**Pull request merged**